apps/framework-cli/src/cli/routines.rs

```diff
@@ -387,6 +387,10 @@ pub async fn start_development_mode(
             &plan_result.target_infra_map,
         )
         .await?;
+        plan_result
+            .target_infra_map
+            .store_in_redis(&redis_client)
+            .await?;
```

Comment:
Are you thinking - Add things to redis and deprecate the clickhouse side once redis is working properly?

Reply:
I like that we have the historical versions in clickhouse, so i guess, might to worth doing, but not urgently.